### DISCUSSION TITLE
Global document throws Fetch PUT error on save

### DISCUSSION THREAD

**Discussion by pbougie** [tinacms/tinacms#4368] [General]
I have a few global JSON documents that can no longer be saved. This is a new issue and no changes have been made to our code. I have updated the TinaCMS libraries and the problem persists. The same configuration allows files 35KB and less to work. However, I have 3 files that are 68KB and larger and these fail with the following error: 

![error](https://github.com/tinacms/tinacms/assets/58460/09e2cc20-f0ac-42b6-8ace-9de8da899df4)

Have any changes or limitations been introduced recently? The problem does not occur in development.

**Comment by kldavis4** [comment]
Is this happening in local dev mode? If not, can you verify whether it happens there or only in the production site?

**Reply by pbougie** [reply]
@kldavis4 No, it does not happen in dev mode. Only in production.

**Comment by kldavis4** [comment]
@pbougie can you try again now and let me know whether it is working?

**Reply by pbougie** [reply]
@kldavis4 Seems to be working again. What was the problem?

**Reply by pbougie** [reply]
@kldavis4 Thank you. Have a great weekend.
